src/Components/ElementContent.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+const ElementContent = ({title, text}) => {
+  return (
+    <STYLED_INFORMATIONS>
```

Comment:
You should write the components name in pascal case: `StyledInformations` instead of `STYLED_INFORMATIONS`.